### PR TITLE
[Test] Verify Prometheus daemon retention_time is adjustable

### PR DESCRIPTION
**Pull request by sayaleeraut**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by sayaleeraut** [comment]
**Tox execution results -**

py3: commands[1]> flake8
py3: commands[2]> isort -c .
Skipped 3 files
py3: commands[3]> black --check --diff .
All done! ✨ 🍰 ✨
1073 files would be left unchanged.
py3: commands[4]> yamllint -d relaxed --no-warnings .
.pkg: _exit> python /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
  py3: OK (28.06=setup[3.88]+cmd[0.50,1.45,1.35,2.71,18.17] seconds)
  congratulations :) (28.08 seconds)

**Review by MohitBis — DISMISSED** [review]
LGTM

**Review by pranavprakash20 — APPROVED** [review]
Lgtm

File: tests/cephadm/test_adjust_prometheus_retention_time.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    installer.exec_command(sudo=True, cmd=cmd)
+
+    # Validate the changes using curl command
+    timeout, interval = 300, 6
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        _retention_time = get_retention_time(installer, prometheus)
+        if _retention_time != retention_time:
+            log.info("retention_time for Prometheus is changed successfully")
+            break
+        else:
```
Comment (vamahaja) [inline tests/cephadm/test_adjust_prometheus_retention_time.py:88]: `else` is not required here.

File: suites/reef/cephadm/tier-2-prometheus-retention-time.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
+#===============================================================================================
+# Test case: Verify Prometheus daemon retention time is adjustable
+#
+# Conf: conf/reef/cephadm/tier-0.yaml
+#===============================================================================================
+tests:
```
Comment (vamahaja) [inline suites/reef/cephadm/tier-2-prometheus-retention-time.yaml:6]: Why separate test suite ? Can't it be part of any existing test suite ?

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/reef/cephadm/tier-2-prometheus-retention-time.yaml (hunk 2 above)
Comment (sayaleeraut) [reply suites/reef/cephadm/tier-2-prometheus-retention-time.yaml:6]: This test case is part of cephadm component and there is no other tier-2 test suite where I can add this Prometheus related test case.

**Comment by vamahaja** [comment]
LGTM, kindly share test execution logs.

**Comment by sayaleeraut** [comment]
Execution logs - http://magna002.ceph.redhat.com/cephci-jenkins/saraut/PR_3763/

**Review by vamahaja — APPROVED** [review]

**Pull request merged**